enterprise/utils.py

```diff
@@ -822,12 +822,12 @@ def is_course_run_upgradeable(course_run):
     return False
 
 
-def is_course_run_published(course_run):
+def is_course_run_unpublished(course_run):
     """
-    Return True if the course run's status value is "published".
+    Return True if the course run's status value is "unpublished".
     """
     if course_run:
-        if course_run.get('status') == 'published':
+        if course_run.get('status') == 'unpublished':
```

Comment:
Why was this causing the issue? Do we not have the correct value of status for published courses?